Tests/Binary/Loader/GridFSLoaderTest.php

```diff
@@ -23,6 +23,10 @@ class GridFSLoaderTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
+        if (!class_exists('Doctrine\ODM\MongoDB\DocumentRepository')) {
+            $this->markTestSkipped('Doctrine ODM not installed');
```

Comment:
Ah, I thought it would be sufficient to check for ODM, as that’s the one that requires ext/mongodb. My idea was to check for the highest abstraction and not all the way down. But I don’t mind adding it.